ui/app/main-template.html

```diff
@@ -0,0 +1,101 @@
+<div class="container" ng-cloak>
+  <div ng-show="isReady">
+    <div class="page-header">
+      <h1>SKY API Authorization Code Flow Tutorial</h1>
```

Comment:
Removed "Blackbaud" from the title due to A-Team's recent requirement that the API be called "SKY API" only.